api/src/main/java/org/openmrs/liquibase/ChangeLogVersionFinder.java

```diff
@@ -88,9 +88,7 @@ public ChangeLogVersionFinder(ChangeLogVersions changeLogVersions) {
 		SortedMap<String, List<String>> changeLogCombinations = new TreeMap<>();
 		
 		for (String snapshotVersion : getSnapshotVersions()) {
-			List<String> changeLogFilenames = new ArrayList<>();
-			
-			changeLogFilenames.addAll(getSnapshotFilenames(snapshotVersion));
+			List<String> changeLogFilenames = new ArrayList<>(getSnapshotFilenames(snapshotVersion));
```

Comment:
does this look good enough to you nowo ? cc @sacull

Reply:
@HerbertYiga I don't think you need to wait for my answer. How do you think about it yourself? Is this PR ready to merge or not yet? ;)